packages/api/README.md

```diff
@@ -951,3 +963,395 @@ Response codes:
+    "publicKeys": [
+        {
+            "$version": "0",
+            "id": 0,
+            "type": 0,
+            "purpose": 0,
+            "securityLevel": 0,
+            "contractBounds": null,
+            "readOnly": false,
+            "data": "Alir4EiGMI/rUrjz1k6s5JE8nQSfTdqaiKIX5spriaEH",
+            "signature": "1f60451588fe72a067daaa0a1ee04279e77ce346128560129162386f76d51eccdc1d88704f2262fe173de57e5598010655d410da94ae2e1cf7086049878b08e966"
+        },
+        {
+            "$version": "0",
+            "id": 1,
+            "type": 0,
+            "purpose": 0,
+            "securityLevel": 2,
+            "contractBounds": null,
+            "readOnly": false,
+            "data": "A+ZoC7Vg5ArbKZprlA09y+Kwjm4aZLyPa8nsPWOGVcNV",
+            "signature": "20066559ccd6cea8ac2d366980f77a94cbfdfbd978803edbf4066f42bc53adcdb51956fb0d3c9cec2012583d17b66456094a8620109d6dae29dc562b2870592940"
+        },
+        {
+            "$version": "0",
+            "id": 2,
+            "type": 0,
+            "purpose": 0,
+            "securityLevel": 1,
+            "contractBounds": null,
+            "readOnly": false,
+            "data": "AjJqjBmhxY0w0ULhE9Dd84HZWmMG9WyexNPLjEgjaFsp",
+            "signature": "1f5bb82721b58d92e67db9fb699ab939ccc4a6d5e2e498e80dfb8a3535c13f571923f045e645a898762f8305a4a2218bfedb060f8a8492c48ae9c96247ce17710b"
+        },
+        {
+            "$version": "0",
+            "id": 3,
+            "type": 0,
+            "purpose": 3,
+            "securityLevel": 1,
+            "contractBounds": null,
+            "readOnly": false,
+            "data": "AlKi0I8pWHHsSwTLC897KJmwsARwLZBYmC3XlxQdUn54",
+            "signature": "2044820dc7651186634326922eda85741bb3f9f005057d94b36845a7edc16ed1df4d5ccabd7e7f003e9c189847fbc06e943252640bc47963c42ae6c0d87b7b506b"
+        }
+    ]
+}
+```
+```
+{
+    "type": 3,
+    "input": "yfcGcMzUw3RANR5f2qYavnxbQobLD93APi",
+    "assetLockProof": {
+        "instantLock": "AQHlBKj77xbL0VC5XRKhuYg3RIqJTVfYQ4EWUtu6lmdVmAEAAADvzDrsdTKnOK8vC7BZ9uTw3xT3+j7YvrBwM8ZIWlCo9FQPCW6UgYX1XXl8oLqo9TYuDj4CgSStivEwPNTdAAAAhNXZYqwU/zhpaCKxSFLtKe+V9bhm8Gxtn533AC1gBoLP32Z0htf3AdPB9yoKI314GM4XsMZECqOB1HWu/tHmZ8c7JVg0Vp8OMShvEWWXbBTpy3vy5aAe8o5AtyLSc5VE",
```

Comment:
There could be instantSend lock or chainlock, we need to document both cases

Reply:
✅